tasks.py

```diff
@@ -57,6 +57,22 @@ def server(host=None, port=5000, debug=True, live=False):
 
 
 @task
+def debug_server(host=None, port=5000, debug=True, live=False):
+    """Run the app server with debugging"""
+
+    def run_app():
+
+        from website.app import init_app
+        app = init_app(set_backends=True, routes=True)
+        settings.API_SERVER_PORT = port
```

Comment:
Can you please explain why you're changing the API server port? This is a different app than the main flask app.